3467/0/about_array_assignment.rb

```diff
@@ -0,0 +1,54 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# :reek:TooManyMethods
+class AboutArrayAssignment < Neo::Koan
+  def test_non_parallel_assignment
+    names = %w[John Smith]
+    assert_equal %w[John Smith], names
+  end
+
+  def test_parallel_assignments
+    first_name = 'John'
+    last_name = 'Smith'
```

Comment:
is it really parallel assignment? Replace with original version and disable rubocop warning for the method.